examples/01_basic_api_key_migration.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Example 1: Basic API Key Migration
+ * 
+ * This example shows how to migrate from direct cURL API calls
+ * or SDK v1 to SDK v2 using API key authentication.
+ */
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use BhrSdk\Client\ApiClient;
+use BhrSdk\ApiException;
+
+// ============================================================================
+// OLD WAY: Direct cURL API calls
+// ============================================================================
+
+function oldWay_getEmployee(string $subdomain, string $apiKey, string $employeeId): array {
+	$url = "https://{$subdomain}.bamboohr.com/v1/employees/{$employeeId}";
```

Comment:
One general piece of feedback - not specific to this function but I'll put it here. 
I do think a lot (maybe most? idk) of people are using curl (or an equivalent).
Some might be using this: https://github.com/BambooHR/bhr-api-php/blob/master/BambooHR/BambooAPI.php
Which has `getDirectory()` and `getEmployee()`, among others, and it might be worth alluding to here and there.